packages/api-plugin-fulfillment-method-pickup-store/src/util/validateOrderMethodsstore.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @summary Sample dummy function validating the data requirements for this metod
+ * @param {Object} context - Context object
+ * @param {Object} commonOrder - Current order which provide available details to perform validation
+ * @param {Object[]} validationResults - Validation results collected till now
+ * @returns {Object[]} validationResults - with the validation details populated
+ */
+export default function validateOrderMethodsstore(context, commonOrder, validationResults = []) {
```

Comment:
This is not correctly camel-cased. If we are doing this because of some string concatenation thing we should def not be doing that

Reply:
Fixed this. Changed the string concatenation logic.

Reply:
placeOrder refactor PR (#6583) will have the related code change